src/tablestore.rs

```diff
@@ -213,17 +205,19 @@ impl TableStore {
                 _ => SlateDBError::ObjectStoreError(e),
             })?;
 
-        self.cache_filter(*id, encoded_sst.filter);
+        self.cache_filter(*id, encoded_sst.info.filter_offset, encoded_sst.filter)
+            .await;
         Ok(SsTableHandle {
```

Comment:
PR description says that SsTableInfo is cached as well, shouldn't there be some change here for that to happen?

Reply:
We shouldn't really need to fetch that from cache. It's always kept in memory for every SST in its `SsTableHandle`.

Reply:
Yes!
I was thinking about the block index. Will check again later.